app/inject/I18nResolverProvider.java

```diff
@@ -32,7 +32,7 @@ public I18nResolverProvider(final Configuration configuration, final ProjectCont
 
     @Override
     public I18nResolver get() {
-        final List<Locale> locales = projectContext.languages();
+        final List<Locale> locales = projectContext.locales();
```

Comment:
We have the same in `UserContext`. What you receive is a list of locales, not of languages. A locale can have regional variations while the language doesn't. I think therefore is more appropriate to call it locales instead of languages. Also, for me `languages` would be more a String that contains the meaning of a language in any form, but in this case is an instance of `Locale`.